draft-js-mention-plugin/src/MentionSuggestions/MentionSuggestions.tsx

```diff
@@ -91,14 +139,18 @@ export class MentionSuggestions extends Component {
       return removeList();
 
     // identify the start & end positon of each search-text
-    const offsetDetails = searches.map(offsetKey => decodeOffsetKey(offsetKey));
+    const offsetDetails = searches.map(offsetKey =>
+      decodeOffsetKey(offsetKey!)
+    );
 
-    // a leave can be empty when it is removed due e.g. using backspace
+    // a leave can be empty when it is removed due event.g. using backspace
```

Comment:
```suggestion
    // a leave can be empty when it is removed due e.g. using backspace
```
Accidental replace probably.